pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -551,9 +533,55 @@ func (cm *containerManagerImpl) Start(node *v1.Node, activePods ActivePodsFunc)
 		}, 5*time.Minute, wait.NeverStop)
 	}
 
+	// Local storage filesystem information from `RootFsInfo` and `ImagesFsInfo` is availalbe at a later time
+	// depending on the time when cadvisor manager updates container stats. Therefore use a go routine to keep
+	// retrieving the information until it is avaialble within 5 minutes timeout.
```

Comment:
I was thinking multiple go routine might running without stops (each time kubelet restarts, a new go rountine for this will be started)